actionpack/lib/action_dispatch/middleware/ssl.rb

```diff
@@ -131,7 +131,7 @@ def redirect_to_https(request)
       end
 
       def redirection_status(request)
-        if request.get? || request.head?
+        if %w[GET HEAD].include?(request.raw_request_method)
```

Comment:
Let's extract this array to an constant.

Reply:
Done!